### PR TITLE
Torsten/timed

### PR DESCRIPTION
**Pull request by torsten-deriv**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ DataDog::DogStatsd - A Perl client for DogStatsd
 
 Statsd: A DogStatsd client (https://www.datadoghq.com)
 
-The difference between [Net::Statsd](https://metacpan.org/pod/Net::Statsd) and this is that it supports tags and namespace.
+The difference between [Net::Statsd](https://metacpan.org/pod/Net%3A%3AStatsd) and this is that it supports tags and namespace.
```
Comment (bruce-binary) [inline README.md:25]: ok... but why though?

**Review by bruce-binary — APPROVED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (torsten-deriv) [reply README.md:25]: not done by me. This happened during `dzil run cpanm -l /home/git/regentmarkets/cpan/local .`

File: lib/DataDog/DogStatsd/Helper.pm [hunk 2]
```diff
@@ -65,9 +96,132 @@ DataDog::DogStatsd::Helper - shortcut/helper for L<DataDog::DogStatsd>
...
+        ...
+    } 'test.timing', {tags => \@tags}, @params;
+
+    DataDog::DogStatsd::Helper->get_dogstatsd;
+    DataDog::DogStatsd::Helper->set_dogstatsd(DataDog::DogStatsd->new(...));
+
 =head1 DESCRIPTION
 
-DataDog::DogStatsd::Helper is a helper for L<DataDog::DogStatsd>, it will reuse the instance of L<DataDog::DogStatsd> for all calls.
+DataDog::DogStatsd::Helper is a helper for L<DataDog::DogStatsd>. It's main purpose
```
Comment (tom-binary) [inline lib/DataDog/DogStatsd/Helper.pm:110]: `Its`

**Pull request merged**